cmd/clusterctl/examples/vsphere/addons.yaml.template

```diff
@@ -6,4 +6,205 @@ metadata:
     storageclass.kubernetes.io/is-default-class: "true"
 provisioner: kubernetes.io/vsphere-volume
 parameters:
-  datastore: ""
\ No newline at end of file
+  datastore: ""
+
+# install weave CNI
```

Comment:
@akutz we might have discussed this earlier, but technically we don't need this either for the `kind` cause since kind installs CNI for us right?

Reply:
That's correct. And even if it doesn't, I'm not sure Weave should ever be present in the addons file.